test/TestServerProjects/url/Generated/Configuration.json

```diff
@@ -17,5 +17,6 @@
   "CredentialHeaderName": "api-key",
   "OperationGroupToResourceType": {},
   "OperationGroupToResource": {},
-  "OperationGroupToParent": {}
+  "OperationGroupToParent": {},
+  "OperationGroupIsTuple": []
```

Comment:
We may want to establish a pattern of default configuration values so we don't serialize empty dictionaries or arrays.

Reply:
Maybe issue it for later, I agree though, our config files are getting bloated.

Reply:
Yeah, not as part of this PR.

Reply:
Issue created [here](https://github.com/Azure/autorest.csharp/issues/1246) to track this item.